cmd/sonobuoy/app/args.go

```diff
@@ -128,6 +130,10 @@ func AddE2EConfigFlags(flags *pflag.FlagSet) *pflag.FlagSet {
 		e2eParallelFlag, defaultMode.E2EConfig.Parallel,
 		"Specify the E2E_PARALLEL flag to the conformance tests. Overrides --mode.",
 	)
+	e2eFlags.String(
+		e2eCustomRegistryFlag, "",
```

Comment:
nit: Should we make this something like `e2eCustomRegistryConfigFlag`? Just to make clear it's expecting a config file and not `gcr.io/heptio-imags` or something?

Reply:
Sure. I struggled with the naming a bit since I didnt want to make it too long/awkward. ala `e2eCustomRegistryFilepathRegistryListFlag` :-p Yours sounds fine to me 👍

Reply:
haha. :1st_place_medal:

Reply:
`e2eRegistryConfigFlag`; by it being a flag its clear enough its custom. The name was getting a bit long :)